storybook/src/stories/person/avatar-component.stories.mdx

```diff
@@ -60,18 +60,35 @@ You can create your own [`<PersonProvider>`](?path=/docs/data-person-provider--p
 
 Name                      | Type                            | Description
 ------------------------- | -----------------------------   | -----------
-`resolve`                 | `PersonResolver`               | Person resolver for a provider.
+`personResolver`          | `PersonResolver`                | Person resolver for a provider.
 
 ### Methods for PersonAvatar
```

Comment:
this are methods for the provider, not the element

Reply:
Yes, they are in Person Provider section, but what I mean with title is these are methods used in provider for person card